docs: clarify how `bumpVersion` for `sbt` works (#10595)
renovatebot/renovate: docs/usage/configuration-options.md

Doing so means that the `package.json` `version` field will mirror whatever the version is that `x` depended on.
Make sure that version is a pinned version of course, as otherwise it won't be valid.

> For `sbt` note that Renovate will update the version string only if it's found in the project's `built.sbt` file.

Reviewer: ```suggestion
For `sbt`, note that Renovate can only update packages that have a version string in their project's `built.sbt` file.
```

I find the original sentence hard to understand. So how about something like this?

I don't know if my suggestion is technically-correct, so please check committing it. 😉

Author: I was following the example in line 182.  And the comma isn't actually necessary: we make a pause when we say it outloud, but we don't write it down.  "For sbt" isn't a subordinate clause.

"can only update packages" makes it sound like Renovate won't do _any_ updates if the version string isn't in that file.  

I think my wording is more correct, in that it is specific to the version string being updated.

How about?
```
For `sbt` note that Renovate will update the version string only for packages that have the version string in their project's `built.sbt` file.
```

Reviewer: I like your new text way better! ✨ 

But put it on one line. We use one line per sentence in our docs. This makes it easier to work with the documentation, as you can move lines around easily. 😉 

```suggestion
For `sbt` note that Renovate will update the version string only for packages that have the version string in their project's `built.sbt` file.
```

---

> I was following the example in line 182. And the comma isn't actually necessary: we make a pause when we say it outloud, but we don't write it down. "For sbt" isn't a subordinate clause.

I do tend to put too many commas in my text, so I'll take your advice on this. 😉
